aws-py-ecs-instances-autoapi/py-ecs-instance/__main__.py

```diff
@@ -0,0 +1,202 @@
+# Deploys ECS with EC2 container instances (as opposed to using Fargate).
+#
+
+"""An AWS Python Pulumi program"""
+import json
+import pulumi
+import pulumi_aws as aws
+
+aws_config = pulumi.Config("aws")
+region = aws_config.get("region") or "us-east-1" 

```

Comment:
Nit: This could also be written as:

```suggestion
region = aws.config.region or "us-east-1" 
```